Gemfile.lock

```diff
@@ -319,7 +319,7 @@ GEM
     dotenv (0.10.0)
     dotenv-rails (0.10.0)
       dotenv (= 0.10.0)
-    dough-ruby (5.12.0.267)
+    dough-ruby (5.12.0.270)
```

Comment:
Can we try to use semantic versioning here, and actually rev MINOR or PATCH?

I almost pulled this into my update for rio because of this.